tripy/tripy/frontend/module/parameter.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, tensor: Any) -> None:
         t = tensor
         # for convenience, this will convert other dlpack-supporting representations too
         if not isinstance(t, Tensor):
-            t = Tensor(t)
+            t = Tensor.create_directly(t)
```

Comment:
Just curious, do we not want type checking here?

Reply:
For internal uses, we probably don't want more registry calls than we have to but there's no inherent reason not to use ordinary constructor calls.

Reply:
Yup, makes sense.

Reply:
That is true, we can call into the registry for this one.